src/main/java/IP/duke/main/Date.java

```diff
@@ -0,0 +1,72 @@
+package IP.duke.main;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the local date format of the given date..
+ * 
+ * @author Gordon Yit
+ * @Version CS2103T Semester 1
+ */
+public class Date {
+    protected LocalDate localDate;
+    private String[] date;
+    
+    /**
+     * Class constructor.
+     * 
+     * @param dateString string specifying a date in the form DD/MM/YYYY.
+     * @throws DateTimeParseException
+     */
+    public Date(String dateString) throws DateTimeParseException {
+        date = dateString.split("/");
+            localDate = LocalDate.parse(String.format("%s-%s-%s", "2021", date[1], date[0]));
+    }
+
+    /**
+     * Returns string format of the datetime.
+     *
+     * @return date in form DD MMM YYYY.
+     */
+    @Override
+    public String toString() {
+        String dateString = localDate.format(DateTimeFormatter.ofPattern("MMM dd"));
+        return dateString;
+    }
+
+    /**
+     * Retrieves the local date object.
+     * 
+     * return the local date of the date due.
```

Comment:
Should there be a `@` symbol in the beginning?
```suggestion
     * @return the local date of the date due.
```